### PR TITLE
Link to the actual messages

### PR DESCRIPTION
**Pull request by dlmccoy**
This commit links the contents of the action div to the actual message in facebook and links the user's name to their profile.

### CODE REVIEW

File: src/Action.jsx [hunk 1]
```diff
@@ -46,26 +46,36 @@ function Action(props) {
...
-              <div className={styles.convName}>{action.thread.name}</div>,
-            ]
-          }
-        </div>
-        <div className={styles.date}>{moment(action.endDate).fromNow()}</div>
-        <div className={styles.typings}>
-          {typings}
+      <a
+        className={cx([
+          styles.actionLink,
```
Comment (alexkirsz) [inline src/Action.jsx]: No need to use `cx` here since we only have one className. `className={styles.actionLink}`

**Comment by alexkirsz** [comment]
Thanks for the PR!

Haven't tested it yet, but looking at the code, I only have [one comment](https://github.com/Morhaus/facebook-sixth-sense/pull/14/files#r66254507).

File: src/Action.jsx (hunk 1 above)
Comment (dlmccoy) [inline src/Action.jsx]: Fixed!

**Comment by alexkirsz** [comment]
I have received a Cease and Desist from Facebook, so I'm closing out this repo.

Sorry!

**Pull request closed without merging**